### ISSUE TITLE
Move all OOTB features from apps to libs. 

### ISSUE THREAD

**Issue by wildone** [websight-io/starter#164]
**Is your feature request related to a problem? Please describe.**
Currently `apps` folder is filled with lots of content that at first glance should be in `libs` folder. This would allow easy ovrrides of UI elements. Is this something possible?

**Comment by paweljozwicki** [comment]
We use absolute paths for UI elements, so moving them won't help, I think.

**Comment by wildone** [comment]
So is pattern then to just to override that paths?

**Comment by paweljozwicki** [comment]
In terms of UI elements, we didn't anticipate an option to overwrite these elements.

I understand that you're in a situation where overwriting certain elements would enable some improvements, am I right? I will initiate a discussion with the team regarding this matter, but I'm concerned that it would have a significant impact on the architecture of the entire solution.

**Comment by wildone** [comment]
Like all the places where code is hard coded to read /apps can just do /mnt/overlay, and the actual code can be in libs? This would make it very clear that all of the code of the OOTB framework is in LIBS and all the custom code is in APPS.

**Comment by paweljozwicki** [comment]
I think it's a good point. We will consider that change, but as I wrote earlier, it will enforce some changes in many places, so I'm not sure when we would do that. 

But you remind us, that there is an additional mechanism provided by Sling to override existing files, so you can always use URL mappings for that:

<img width="2298" alt="Screenshot 2023-05-26 at 12 26 08" src="https://github.com/websight-io/starter/assets/109656376/b6672e00-5401-4a58-ad31-e38752461699">

I hope it would unblock you.

**Comment by wildone** [comment]
Yes URL mapping will work short term, right now I just make the paths under apps to force my updates.
